proposals/blogProposal.md

```diff
@@ -0,0 +1,5 @@
+The Web of Things standardization activities are now visible(?) in our new animation video that we have been working on for the last couple of months. You can see how any web developer can implement Internet of Things solutions while being interoperable and not locked-in to framework. Additionally, the W3C WoT allows the experience of the developers to be transferred to different application domains such as agriculture, building or industrial automation, smart home or more!
+
+-- To be decided
+
+We would like to thank the W3C Accesibility team for their feedback, as well as Siemens AG and Intel Inc. for the support and the team at WebCastor for quick iteration cycle to come up with an amazing video!you
```

Comment:
"to come up with an amazing video!you" --> "to come up with our amazing video. [See your self!](TODO-Link)"

Reply:
Done